app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -334,28 +275,40 @@ public AddWitnessResult addWitness(Lao lao, String witness) {
      */
     public void activateTestingValues() {
         currentLao = new Lao("LAO I just joined", person.getId());
-        dummyLaoEventMap();
+        dummyLaos();
+    }
+
+    private List<Event> dummyEvents(String laoId) {
```

Comment:
Could you swap the methods `dummyEvents` and `dummyLao`?

Reply:
Done